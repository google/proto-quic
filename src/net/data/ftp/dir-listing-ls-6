total 14
dr-xr-xr-x   7 0        1            512 Jun 19  2006 .
dr-xr-xr-x   7 0        1            512 Jun 19  2006 ..
dr-xr-xr-x   2 0        1            512 Mar 24  2003 bin
dr-xr-xr-x   2 0        1            512 Mar 24  2003 etc
dr-xr-xr-x  12 0        0            512 Apr  7  2009 pub
dr-xr-xr-x   3 0        1            512 Mar 24  2003 usr
