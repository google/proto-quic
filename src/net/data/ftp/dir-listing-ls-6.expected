d
.
-1
2006
6
19
0
0

d
..
-1
2006
6
19
0
0

d
bin
-1
2003
3
24
0
0

d
etc
-1
2003
3
24
0
0

d
pub
-1
2009
4
7
0
0

d
usr
-1
2003
3
24
0
0
